### PR TITLE
Add new Span node design

### PR DESCRIPTION
**Pull request by jorgeepc**
This PR adds the latest `Span Node` design.

## Changes

- Span Node

## Fixes

- #804 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Screenshot

Screen Shot 2022-07-01 at 12 20 20

### CODE REVIEW

File: web/src/models/Span.model.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ const Span = ({id = '', attributes = {}, startTime = 0, endTime = 0, parentId =
 
   const name = attributes[Attributes.NAME] || '';
   const kind = SpanKindList.find(spanKind => spanKind === attributes[Attributes.KIND]) || SpanKind.INTERNAL;
-  const duration = Number(attributes[Attributes.TRACETEST_SPAN_DURATION]) || 0;
+  const duration = attributes[Attributes.TRACETEST_SPAN_DURATION] || '0ns';
```
Comment (jorgeepc) [inline web/src/models/Span.model.ts:45]: The duration now is a string. The BE change broke the timeline graph. We need to figure out the best approach  for the duration.

**Review by xoscar — APPROVED** [review]
Great work man

File: web/src/components/Diagram/components/Timeline/Timeline.tsx [hunk 2]
```diff
@@ -184,7 +184,7 @@ const Timeline = ({affectedSpans, spanList, selectedSpan, onSelectSpan}: IDiagra
       .append('text')
       .attr('class', 'span-duration')
       .attr('y', 30)
-      .text((d: any) => `${d.data.data?.duration} ms`);
+      .text((d: any) => d.data.data?.duration);
```
Comment (xoscar) [inline web/src/components/Diagram/components/Timeline/Timeline.tsx:187]: This ticket  https://github.com/kubeshop/tracetest/issues/839 is going to introduce more of the changes related to the duration

Comment (jorgeepc) [reply web/src/components/Diagram/components/Timeline/Timeline.tsx:187]: Cool, with this PR I believe the graphs are ok.

**Pull request merged**